src/Uno.UI/Extensions/PermissionsHelper.Android.cs

```diff
@@ -19,11 +19,185 @@ public static void Initialize()
 					.CreateAsync<string, bool>(TryGetPermissionCore)
 					.LockInvocation(InvocationLockingMode.Share);
 
-			Windows.Extensions.PermissionsHelper.Initialize(getPermission, CheckPermission);
-		}
+			Windows.Extensions.PermissionsHelper.Initialize(getPermission, CheckPermission, MissingPermissions, TryAskPermissionAsync);
+		}
+
+		private static int _permissionRequest;
+		#region "submethods for MissingPermissions"
+		private static IList<string> GetManifestPermissions()
+		{
+			// get all permission declared in Manifest (can be null)
+			Android.Content.Context context = Android.App.Application.Context;
+			Android.Content.PM.PackageInfo packageInfo =
+				context.PackageManager.GetPackageInfo(context.PackageName, Android.Content.PM.PackageInfoFlags.Permissions);
+			return packageInfo?.RequestedPermissions;
+		}
+
+		private static bool AreAllPermissionDeclared(IList<string> manifestPermissions, string[] requiredPermissions)
+		{
+			// check if all requiredPermissions are declared in Manifest 
+			if (requiredPermissions is null)
+			{
+				return true;    // no required permissions, so - everything is OK
+			}
+
+			foreach (string permission in requiredPermissions)
+			{
+				if (!manifestPermissions.Any(p => p.Equals(permission, StringComparison.OrdinalIgnoreCase)))
+				{
+					return false;
+				}
+			}
+
+			return true;
+
+		}
+
+		private static List<string> PermissionWeWant(IList<string> manifestPermissions, string[] requiredPermissions, string[] optionalPermissions)
+		{
+			// prepare list of all permissions
+			var allPermissions = new List<string>();
+			if (requiredPermissions != null)
+				allPermissions.AddRange(requiredPermissions.ToList());
+
+			// add all optional permission, found in Manifest
+			if (optionalPermissions != null)
+			{
+				foreach (string permission in optionalPermissions)
+				{
+					if (manifestPermissions.Any(p => p.Equals(permission, StringComparison.OrdinalIgnoreCase)))
+					{
+						allPermissions.Add(permission);
+					}
+				}
+			}
+
+			return allPermissions;
+		}
+
+		private static List<string> PermissionToAsk(IList<string> allPermissions)
+		{// return list of permission, with all permission from input list that are not already granted

```

Comment:
This is a comment that goes into XML comments above the method

Reply:
a) how can I prevent it?
b) this method is `private`, so any comments could be displayed only within this file (class), so I guess it is not important

Reply:
Sure, but that's this project's standard way to document methods. If the comment is required to explain what the method does, it has its place there.

Reply:
So, where I should place this comment?

Reply:
So, where I should place this comment?

Reply:
Place it as an XML comment, as I mentioned above, that's the way to document methods.

Reply:
Oh, misunderstanding...
I thought that this comment, as it is, is placed into comments visible in editor as tooltip, and that you want it to not be placed into tooltip :)

This method is private, so it should not be called from any other place than main method, GetMissingPermissions()... and in current version these method is converted to one-line LINQ expression in GetMissingPermissions method :)